src/Response/GetSimilarResponse.php

```diff
@@ -33,17 +33,12 @@ static function ($item) {
                 array_values(
                     array_filter(
                         $responseUids,
-                        static fn ($item) => $item !== $originalUid,
+                        static fn($item) => $item !== $originalUid,
                     )
                 )
             );
         } catch (\Throwable $e) {
             throw new MalformedResponse($e->getMessage(), $response, $e);
         }
     }
-
-    public function getUids(): array
-    {
```

Comment:
Do we really want to change response method? It will demand changes in usage in other projects

Reply:
It's php feature, why shouldn't we using it? It's just change of getter to direct property.

Reply:
Ok, so it will be version 3.0.0?

Reply:
Yes, mainly because of dropping support for symfony < 6, and php < 8.2.